### PR TITLE
Update travis config

### PR DESCRIPTION
**Pull request by Rafiot**
* Run tox with python3.6 only
* Update libarchive library
* Fix tests validation.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,15 +16,15 @@ env:
...
+        wget http://libarchive.org/downloads/libarchive-3.3.2.tar.gz &&
+        tar -xf libarchive-3.3.2.tar.gz && cd libarchive-3.3.2 &&
         ./configure --prefix=/opt/python-libarchive-c --disable-bsdcpio --disable-bsdtar &&
         make && sudo make install && cd .. ;
     fi"
 
 install: pip install tox
 
-script: tox
+script: tox -e py36
```
Comment (Changaco) [inline .travis.yml]: That's not acceptable.

Comment (Rafiot) [reply .travis.yml]: Why? The current travis config is expecting python 3.6.

Comment (Changaco) [reply .travis.yml]: The current config isn't limited to 3.6, it also tests against 2.7, 3.4 and 3.5.

**Comment by Changaco** [comment]
Thanks @Rafiot.

**Comment by Rafiot** [comment]
Thank you, sorry, I totally forgot about it...

**Pull request merged**